embed-code-go/fragmentation/fragment_file.go

```diff
@@ -36,8 +36,8 @@ import (
 //
 // Configuration — a configuration for embedding.
 type FragmentFile struct {
-	CodeFile      string                     
-	FragmentName  string                     
+	CodeFile      string
```

Comment:
Let me comment on the description of this field. When we say "relative", it would be good to know what it is relative to.